python/row_by_row_modification.yaml

```diff
@@ -0,0 +1,35 @@
+    Case 1:
```

Comment:
It's OK to name this one as the Q617 specific case and provide a file snippet that allows a user to expand it.